appcenter-analytics/react-native.config.js

```diff
@@ -1,7 +1,11 @@
+const path = require('path');
+
 module.exports = {
   dependency: {
     platforms: {
-      ios: {},
+      ios: {
+        podspecPath: path.join(__dirname, 'ios', 'appcenter-analytics.podspec')
```

Comment:
Does this work with the issue I mentioned in [my previous comment](https://github.com/microsoft/appcenter-sdk-react-native/pull/669/files#r315040070)? Did you test with a blank RN 0.60 app?

Reply:
[Answered.](https://github.com/microsoft/appcenter-sdk-react-native/pull/669#discussion_r315108376)